src/app/layout.tsx

```diff
@@ -15,8 +19,9 @@ export default function RootLayout({
   children: React.ReactNode;
 }>) {
   return (
-    <html lang="en">
-      <body className={inter.className}>{children}</body>
+    <html suppressHydrationWarning={true} lang="en">
```

Comment:
Why are we suppressing hydration warning?


If there is an issue with hydration, use a solution from here if possible?


[Hydration Error](https://nextjs.org/docs/messages/react-hydration-error)